README.md

```diff
@@ -378,6 +378,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [essarrgee](https://github.com/essarrgee)
   - [Euhid Aman](https://github.com/euhidaman)
   - [Evandro Rodrigues](https://github.com/evnrodr)
+  - [Edilson Matola](https://github.com/edilsonmatola)
```

Comment:
Please add your name in line 374, after Edgar Ssenalo.

Reply:
@Panquesito7 Alright.